README.md

```diff
@@ -18,8 +18,12 @@ on the command-line, or set the following environment variables connect your
 node to [app.tenzir.com](app):
 
 ```bash
-export TENZIR_PLUGINS__PLATFORM__API_KEY='<api-key>'
-export TENZIR_PLUGINS__PLATFORM__TENANT_ID='<tenant-id>'
+export TENZIR_TOKEN='<token>'
+
+# This one is only necessary if you're using a self-hosted version of the
+# Tenzir Platform, e.g., because you're a Developer at Tenzir or a user of the
+# Tenzir Sovereign Edition.
```

Comment:
We could probably drop the second half of the sentence.

Reply:
Will do.

Reply:
Done in https://github.com/tenzir/example-plugin/compare/8d2f47b67ff89da489d28d02515e283adc124ce9..6aa9d7336d25ebd8e201129ca1989fe8b3c172bc